src/main/java/de/wejago/hichi2influx/service/SensorMessageSubscriber.java

```diff
@@ -14,21 +16,35 @@
 @Slf4j
 @Service
 @RequiredArgsConstructor
-    public class SensorMessageSubscriber implements IMqttMessageListener {
-        private final ObjectMapper objectMapper;
+public class SensorMessageSubscriber implements IMqttMessageListener {
+    private final ObjectMapper objectMapper;
+    private final InfluxDbRepository influxDbRepository;
 
-        private final InfluxDbRepository influxDbRepository;
-
-        @Override public void messageArrived(String s, MqttMessage mqttMessage) {
-            try {
-                String receivedMessage = new String(mqttMessage.getPayload());
-                log.info("Received message: " + receivedMessage);
-                if(receivedMessage.contains("1_8_0")) {
-                    SensorEntry sensorEntry = objectMapper.readValue(receivedMessage, SensorEntry.class);
-                    influxDbRepository.writePoint(sensorEntry);
-                }
-            } catch (JsonProcessingException e) {
-                log.error("objectMapper JsonProcessingException" + e);
+    @Override public void messageArrived(String s, MqttMessage mqttMessage) {
```

Comment:
method name doesn't really fit to the businnes logic.

Reply:
It is specified from the interface!